staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -3153,6 +3153,22 @@ const (
 	FSGroupChangeAlways PodFSGroupChangePolicy = "Always"
 )
 
+// PodSELinuxRelabelPolicy holds policies that will be used for applying SELinux labels to a volume
+// when volume is mounted.
+type PodSELinuxRelabelPolicy string
+
+const (
+	// SELinuxRelabelOnVolumeMount indicates that, if supported by the storage backend, SELinux
+	// label will be assigned during mount of pod's volumes as a mount option. This can skip
+	// potentially long recursive SELinux label change performed by container runtime when starting
+	// a pod.
```

Comment:
Why would you not always set this? Add the drawbacks to the doc comment.

Reply:
It's... complicated. I added more description there, however, I was not able to avoid mentioning Linux kernel.

Reply:
What is the tradeoff when you skip this?  I.e. you talk about a benefit (performance) but you don't present what you lose (tim's question).  I would guess that you are not secure.  If you're not secure, why.  Is there a link somewhere to the presentation of why relabel is not desirable that we can copy?

Reply:
Sorry, I updated only storage/v1, forgetting core/v1.

Added "As tradeoff, all pods that simultaneously access the volumes must run with the same SELinux context to be able to access data on them, even if they use just a subPath of them."

The goal is to say it's safe, unless you use a single volume with two pods with different SELinux contexts, each using a different subpath of the volume. This was working before and it is working even now with policy "Always".